content/pt-br/docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -0,0 +1,176 @@
+{{< /note >}}
+
+Se você estiver executando o CoreDNS como um Deployment, ele geralmente será exposto como um Serviço do Kubernetes com o endereço de IP estático.
+O kubelet passa informações de resolução de DNS para cada contêiner com a flag `--cluster-dns=<dns-service-ip>`.
+
+Os nomes DNS também precisam de domínios. Você configura o domínio local no kubelet com a flag `--cluster-domain=<default-local-domain>`.
+
+O servidor DNS suporta pesquisas de encaminhamento (registros A e AAAA), pesquisas de porta (registros SRV), pesquisas de endereço de IP reverso (registros PTR) e muito mais. Para mais informações, veja [DNS para Serviços e Pods](/docs/concepts/services-networking/dns-pod-service/).
+
+Se a `dnsPolicy` de um Pod estiver definida como `default`, ele herda a configuração de resolução de nome do nó em que o Pod é executado. A resolução de DNS do Pod deve se comportar da mesma forma que o nó.
+Veja [Problemas conhecidos](/docs/tasks/administer-cluster/dns-debugging-resolution/#known-issues)..
+
+Se você não quiser isso, ou se quiser uma configuração de DNS diferente para os pods, pode usar a flag `--resolv-conf` do kubelet. Defina essa flag como "" para impedir que os Pods herdem a configuração do DNS. Defina-a como um caminho de arquivo válido para especificar um arquivo diferente de `/etc/resolv.conf` para a herança de DNS.
+
+## CoreDNS
+
+CoreDNS é um servidor oficial de DNS de propósito geral que pode atuar como DNS do cluster,
+cumprindo com as [especificações DNS](https://github.com/kubernetes/dns/blob/master/docs/specification.md).
+
+### CoreDNS ConfigMap options
+
+CoreDNS é um servidor DNS que é modular e plugável, com plugins que adicionam novas funcionalidades.
+O servidor CoreDNS pode ser configurado por um [Corefile](https://coredns.io/2017/07/23/corefile-explained/),
+que é o arquivo de configuração do CoreDNS. Como administrador de cluster, você pode modificar o
+{{< glossary_tooltip text="ConfigMap" term_id="configmap" >}} para o arquivo Corefile do CoreDNS para
+mudar como o descobrimento de serviços DNS se comporta para esse cluster.
+
+Em Kubernetes, o CoreDNS é instalado com a seguinte configuração padrão do Corefile:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: coredns
+  namespace: kube-system
+data:
+  Corefile: |
+    .:53 {
+        errors
+        health {
+            lameduck 5s
+        }
+        ready
+        kubernetes cluster.local in-addr.arpa ip6.arpa {
+            pods insecure
+            fallthrough in-addr.arpa ip6.arpa
+            ttl 30
+        }
+        prometheus :9153
+        forward . /etc/resolv.conf
+        cache 30
+        loop
+        reload
+        loadbalance
+    }
+```
+
+A configuração do Corefile inclui os seguintes [plugins](https://coredns.io/plugins/) do CoreDNS:
+
+* [errors](https://coredns.io/plugins/errors/): Erros são registrados para stdout.
+* [health](https://coredns.io/plugins/health/): A saúde do CoreDNS é reportada para
```

Comment:
"saúde" is a bit strange when used in the context of inanimate things. I suggest using "integridade" instead.
I also suggest you write "reportada em", instead of "para".